diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x 'Makefile.in*' -x configure freeglut-2.4.0-orig/configure.ac freeglut-2.4.0/configure.ac
--- freeglut-2.4.0-orig/configure.ac	2005-06-09 20:05:33.000000000 -0500
+++ freeglut-2.4.0/configure.ac	2006-05-12 17:09:31.285750000 -0500
@@ -4,7 +4,7 @@
 AM_INIT_AUTOMAKE
 AC_PREREQ([2.56])
 AC_CONFIG_SRCDIR([AUTHORS])
-AC_CONFIG_HEADERS([config.h])
+AM_CONFIG_HEADER([config.h])
 
 # Compatibility hack for older autoconf versions
 m4_ifdef([AS_HELP_STRING], [], [AC_DEFUN([AS_HELP_STRING], [AC_HELP_STRING($][@)])])
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x 'Makefile.in*' -x configure freeglut-2.4.0-orig/include/GL/freeglut_std.h freeglut-2.4.0/include/GL/freeglut_std.h
--- freeglut-2.4.0-orig/include/GL/freeglut_std.h	2005-01-01 10:09:38.000000000 -0600
+++ freeglut-2.4.0/include/GL/freeglut_std.h	2006-05-12 18:14:45.567000000 -0500
@@ -35,7 +35,7 @@
 /*
  * Under windows, we have to differentiate between static and dynamic libraries
  */
-#if defined(_MSC_VER) || defined(__CYGWIN__) || defined(__MINGW32__)
+#if defined(_MSC_VER) || defined(__MINGW32__)
 
 /* #pragma may not be supported by some compilers.
  * Discussion by FreeGLUT developers suggests that
@@ -183,7 +183,7 @@
  *
  * Steve Baker suggested to make it binary compatible with GLUT:
  */
-#if defined(_MSC_VER) || defined(__CYGWIN__) || defined(__MINGW32__)
+#if defined(_MSC_VER) || defined(__MINGW32__)
 #   define  GLUT_STROKE_ROMAN               ((void *)0x0000)
 #   define  GLUT_STROKE_MONO_ROMAN          ((void *)0x0001)
 #   define  GLUT_BITMAP_9_BY_15             ((void *)0x0002)
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x 'Makefile.in*' -x configure freeglut-2.4.0-orig/src/Makefile.am freeglut-2.4.0/src/Makefile.am
--- freeglut-2.4.0-orig/src/Makefile.am	2005-04-26 08:38:27.000000000 -0500
+++ freeglut-2.4.0/src/Makefile.am	2006-05-12 17:59:52.457625000 -0500
@@ -42,7 +42,7 @@
 # Additional linker flags
 #	 
 lib@LIBRARY@_la_LIBADD = $(LIBM) $(X_LIBS) -lGL -lGLU -lXext -lX11 $(LIBXXF86VM)
-lib@LIBRARY@_la_LDFLAGS = -version-info 11:0:8
+lib@LIBRARY@_la_LDFLAGS = -version-info 11:0:8 -no-undefined
 lib@LIBRARY@_la_CFLAGS = $(X_CFLAGS)
 
 INCLUDES = -I$(top_srcdir)/include
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x 'Makefile.in*' -x configure freeglut-2.4.0-orig/src/freeglut_internal.h freeglut-2.4.0/src/freeglut_internal.h
--- freeglut-2.4.0-orig/src/freeglut_internal.h	2005-06-02 17:39:32.000000000 -0500
+++ freeglut-2.4.0/src/freeglut_internal.h	2006-05-12 17:36:05.629500000 -0500
@@ -42,7 +42,7 @@
 #   define  TARGET_HOST_UNIX_X11    0
 #   define  TARGET_HOST_WIN32       0
 #   define  TARGET_HOST_WINCE       1
-#elif defined(_MSC_VER) || defined(__CYGWIN__) || defined(__MINGW32__)
+#elif defined(_MSC_VER) || defined(__MINGW32__)
 #   define  TARGET_HOST_UNIX_X11    0
 #   define  TARGET_HOST_WIN32       1
 #   define  TARGET_HOST_WINCE       0
